build/Targets/VSL.Versions.targets

```diff
@@ -56,7 +59,7 @@
     <When Condition="'$(OfficialBuild)' == 'true' OR '$(SignedBuild)' == 'true'">
       <PropertyGroup>
         <AssemblyVersion>$(RoslynSemanticVersion).0</AssemblyVersion>
-        <BuildVersion>$(RoslynSemanticVersion).$(BuildNumberPart1)$(BuildNumberPart2)</BuildVersion>
+        <BuildVersion>$(RoslynSemanticVersion).0</BuildVersion>
```

Comment:
That was the case (on MBv2) before my original change (unless a build number was manually specified). On MBv1, the existing setup we had would also have broken next year (2017), as they were in the format of ymmdd (so 60615).